### PR TITLE
OD-340 [Fix] Split row data by commas and check intersection for Is one of

### PR DESCRIPTION
**Pull request by ivandevupp**
@sofiiakvasnevska
## Issue
OD-340 https://weboo.atlassian.net/browse/OD-340
## Description
Split row data by commas and check intersection for Is one of
## Screenshots/screencasts
https://monosnap.com/file/AZq0twTFae3QHxC2gBkTu19ErI4AiP
## Backward compatibility
This change is fully backward compatible.
## Reviewers 
@upplabs-alex-levchenko @AndrRyaz @YaroslavOvdii

### CODE REVIEW

File: js/utils.js [hunk 1]
```diff
@@ -500,7 +500,7 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
         }
 
         if (condition === 'oneof') {
-          return splitByCommas(filter.value).includes(rowData);
+          return Boolean(_.intersectionWith(splitByCommas(filter.value), splitByCommas(rowData), _.isEqual).length);
```
Comment (tonytlwu) [inline js/utils.js:503]:
```suggestion
          return !!_.intersectionWith(splitByCommas(filter.value), splitByCommas(rowData), _.isEqual).length;
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/utils.js (hunk 1 above)
Comment (ivandevupp) [reply js/utils.js]: FIxed.

Comment (ivandevupp) [reply js/utils.js]: Also, added fix for filter values in double quotes, they didn't work as expected because of comma splitting of the row data.

**Review by tonytlwu — CHANGES_REQUESTED** [review]
This is not necessary. The behaviour outlined in https://weboo.atlassian.net/browse/OD-340 is expected and not a bug. The code you added below is in fact a different kind of logic entirely. It's what we would consider **Contains one of**, which is something we're planning to have in the backlog cc @sofiiakvasnevska 

```js
!!_.intersectionWith(splitByCommas(filter.value), splitByCommas(rowData), _.isEqual).length
```

See https://weboo.atlassian.net/browse/OD-340?focusedCommentId=15448 for more.

**Pull request closed without merging**